File: sprixel/misc/quote__S_Double_Double.js

// 
Act.types.quote__S_Double_Double = function(){
    //throw ;
    //throw keys(this.node);
    do { switch(this.phase) {
    case 0:
        this.phase = 1;
        return this.next = new Act(this.node.nibble, this);
    case 1:
        this.result = this.next.result;
    default: break;
    } } while (false);
    this.next = null;
    return this.invoker;
};
1;

File: vill/src/quote__S_Double_Double.cpp
// quote__S_Double_Double.cpp
#include "villCompiler.h"

Value *
quote__S_Double_Double( struct villCompiler * vill, struct graph_node * node ) {
  Value * result = NULL;
  vill -> debug_flags & DEBUG_USER && fprintf( stderr, "<quote__S_Double_Double>" );
  graph_node * nibble = ast_mapentry( node, "nibble" );
  function_pointer Codegen;
  Codegen = (function_pointer) nibble -> data;
  assert( Codegen != NULL );
  result = Codegen( vill, nibble );
  vill -> debug_flags & DEBUG_USER && fprintf( stderr, "</quote__S_Double_Double>" );
  return result;
}
